Source/Documentation/Manual/timetable.rst

```diff
@@ -441,6 +442,18 @@ Special rows
   train is removed from the activity after it has terminated.
 
   *The #dispose row presently does not affect the end of the run for the player train*. 
+  
+- ``#briefing`` row
+
+  The ``#briefing`` row is optional and contains text which describes the train
+  operation for the user. This text appears in the Open Rails main window along
+  with description of the route and the loco.
```

Comment:
The first column can also set a briefing that describes the entire timetable. This should be documented.

Reply:
Another good catch. I thought of this myself, but only after I had submitted the Pull Request.

Will do.